products/surveys/backend/max_tools.py

```diff
@@ -0,0 +1,196 @@
+"""
+MaxTool for AI-powered survey creation.
+"""
+
+from typing import Any
+from pydantic import BaseModel, Field
+from langchain_core.prompts import ChatPromptTemplate
+from langchain_openai import ChatOpenAI
+
+from ee.hogai.tool import MaxTool
+from posthog.models import Team
+from .prompts import SURVEY_CREATION_SYSTEM_PROMPT
+from .survey_schema import SurveyCreationOutput
+
+
+class SurveyCreatorArgs(BaseModel):
+    instructions: str = Field(description="Natural language description of the survey to create")
+
+
+class SurveyCreatorTool(MaxTool):
+    name: str = "create_survey"
+    description: str = "Create and optionally launch a survey based on natural language instructions"
+    thinking_message: str = "Creating your survey..."
```

Comment:
No need to add the three dots "..." as they are added automatically in a later step.